src/main/java/biomesoplenty/api/item/BOPItems.java

```diff
@@ -99,11 +99,14 @@
     public static Item flower_basket;
     public static Item jar_empty;
     public static Item jar_filled;
-    public static Item sand_bucket;
-    public static Item honey_bucket;
-    public static Item blood_bucket;
-    public static Item poison_bucket;
-    public static Item hot_spring_water_bucket;
+    
+    /*
+     *  public static Item sand_bucket;
+     *  Use net.minecraftforge.fluids.UniversalBucket.getFilledBucket(
+     *          net.minecraftforge.common.ForgeModContainer.getInstance().universalBucket,
+     *          biomesoplenty.common.fluids.QuicksandFluid.instance
+     *      )
+     */
     
```

Comment:
Just one last minor thing, could you remove this as well, since it's a bit weird and out-of-place now. Then it's ready to merge